data/sql_updates/large_aggregates.sql

```diff
@@ -123,7 +123,43 @@ party_totals as (
     group by
         month,
         year
-), -- merge
+),
+communication_totals as (
+  select
+    get_cycle(rpt_yr) as cycle,
+    f.rpt_yr as year,
+    extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as month,
+    sum(coalesce(ttl_communication_cost ,0)) as comms_totals
+  from public.fec_vsum_f7_vw f
```

Comment:
Using fec_fitem_f76_vw to calculate communication totals yielded the following results:
```
select get_cycle(rpt_yr) as cycle, sum(communication_cost)
from disclosure.fec_fitem_f76 where rpt_yr between 2007 and 2016 group by cycle order by cycle;

 cycle |     sum     
-------+-------------
  2008 | 64228059.97
  2010 | 24907901.27
  2012 | 26193530.72
  2014 | 10743452.15
  2016 | 19964137.24
```

These are notably different than from the other table (public.fec_vsum_f7_vw).  Which can be seen directly above by Paul's comment.